docs/releases.md

```diff
@@ -1,4 +1,79 @@
-## Branch strategy
+# Release Process
+
+#### 1. Maintainers quorum
+
+* A majority of maintainers must agree on a single commit SHA for the release. The agreement should be documented in the release notes.
+* The maintainers will match the commit SHA to the release `major.minor.patch` in the release notes
+* The maintainers will agree on a changelog proposal that MUST be in a PR prior to the release.
+* The main `REAMDE.md` must also reflect the new release. Ideally this should be in the same PR.
+
+#### 2. Merge the release notes
+
+* `HISTORY.md` serves as our official changelog.
+* Before the release, the maintainers had opened a PR with the proposed changelog. This should now be merged.
+
+
+#### 3. Release kops to github
+
+*  Darwin binary (generated)
+*  Linux binary (generated)
+*  Source code (zip)
+*  Source code (tar.gz)
+
+#### 4. Release kops to homebrew
+
+* Following the [documentation](development/homebrew.md) we must release a compatible homebrew formulae with the release.
+* This should be done at the same time as the release, and we will iterate on how to improve timing of this.
```

Comment:
I think ultimately `kops` maintainers are responsible for this, we can push on him to help, but at the end of the day it's on us to make sure this stay updated. If we are eventually scripting this process, we need it here.